test/src/test/java/com/alibaba/nacos/test/core/JacksonUtilsTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.test.core;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
+ */
+public class JacksonUtilsTest {
+
+	@Test
+	public void testToJson() {
+		Map<String, Map<String, Object>> map = new HashMap<>();
+		Map<String, Object> data = new HashMap<>();
+		data.put("test-1", LocalDateTime.of(2021, 10, 24, 10, 24));
+		data.put("test_2", LocalDateTime.of(2021, 6, 1, 6, 1));
+		map.put("global", data);
+
+		String json = com.alibaba.nacos.common.utils.JacksonUtils.toJson(map);
+		Assert.assertEquals("{\"global\":{\"test-1\":[2021,10,24,10,24],\"test_2\":[2021,6,1,6,1]}}", json);
+	}
```

Comment:
Please keep the original test and add your test

Reply:
I can do this, but I still find test cases without assertions meaningless

Reply:
Add the assert is right. But I have some problem for this case.

For this test, I think we should test the object to json and json to object by the Util. But this case only do some test for serialization of LocalDateTime which I remember nacos do not serialize.

I has no idea the original author add this cases. But I think if you want to fix this case and make it meaningful, you can add case for every line of test class without any more dependency.